Sources/UILocalNotification+Sweet.swift

```diff
@@ -24,9 +24,9 @@ public extension UILocalNotification {
 
     public static func find(_ id: String) -> UILocalNotification? {
         let notifications = UIApplication.shared.scheduledLocalNotifications ?? [UILocalNotification]()
-        let notification = notifications.filter { $0.userInfo?[UILocalNotification.idKey] as? String == id }
+        let filteredNotifications = notifications.filter { $0.userInfo?[UILocalNotification.idKey] as? String == id }
```

Comment:
👏 

Can you also use this naming? That would clarify it even more in my opinion
```
let notification = notifications.filter { notification in notification.userInfo?[UILocalNotification.idKey] as? String == id }
```